### PR TITLE
style: add variable for aliyun theme

### PR DESCRIPTION
**Pull request by hi-zhaoyeqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (add variable for aliyun theme)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |add variable for aliyun theme|
| 🇨🇳 Chinese |为阿里云主题新增部分新变量|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/drawer/style/drawer.less [hunk 1]
```diff
@@ -134,8 +134,8 @@
     margin: 0;
     color: @heading-color;
     font-weight: 500;
-    font-size: @font-size-lg;
-    line-height: 22px;
+    font-size: @drawer-header-title-font-size;
+    line-height: @drawer-header-title-line-height;
```
Comment (afc163) [inline components/drawer/style/drawer.less:138]:
```suggestion
    font-size: @drawer-title-font-size;
    line-height: @drawer-title-line-height;
```

**Comment by afc163** [comment]
一次性加了好多，分组件提吧，不然不好 review。

File: components/page-header/style/index.less [hunk 2]
```diff
@@ -6,7 +6,7 @@
 .@{pageheader-prefix-cls} {
   .reset-component();
   position: relative;
-  padding: @page-header-padding-vertical @page-header-padding;
+  padding: @page-header-padding-top @page-header-padding @page-header-padding-bottom;
```
Comment (vthinkxie) [inline components/page-header/style/index.less:9]: 这部分不应当修改

File: components/tooltip/style/index.less [hunk 3]
```diff
@@ -93,7 +93,7 @@
   &-placement-top &-arrow,
   &-placement-topLeft &-arrow,
   &-placement-topRight &-arrow {
-    bottom: @tooltip-distance - @tooltip-arrow-rotate-width;
```
Comment (vthinkxie) [inline components/tooltip/style/index.less:96]: 不要修改 bottom 与 top，增加 border 可以使用 `box-shadow: 0 0 0 1px #dedede, 0 1px 4px 0 rgb(0 0 0 / 15%)` 多重阴影的方式来写，不影响 box 模型

**Pull request closed without merging**